examples/react-native/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "main": "node_modules/expo/AppEntry.js",
```

Comment:
Is this required? Can we remove it?

Reply:
I believe this is required for Metro to work. It's comes by default with the blank Expo project.